R/cluster_sce.R

```diff
@@ -0,0 +1,27 @@
+#' Perform clustering on a SingleCellExperiment object
+#'
+#' @param sce SingleCellExperiment object that requires clustering
+#' @param pca_name The name used to access the PCA results
+#' @param BLUSPARAM A BlusterParam object specifying the clustering algorithm to
+#' use - can be `KmeansParam` or `NNGraphParam`
+#' @param cluster_name The type of clustering method to be performed - can be
```

Comment:
I think I would change this variable to reflect that it's just being used to add a column with clustering results to the SCE. Maybe `cluster_column_name`?

Reply:
cluster_column_prefix maybe? Again here we probably don't need to have the function say what is allowed.